subprojects/mate-submodules.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-git]
+directory =mate-submodules
+url=https://github.com/mate-desktop/mate-submodules.git
+push-url=ssh://git@github.com:mate-desktop/mate-submodules.git
+revision=meson
```

Comment:
It works, but depends https://github.com/mate-desktop/mate-submodules/pull/15, so here use the `meson` branch of `mate-submodules`.

And we use meson's subprojects feature, so after switching to use meson build system only, we can drop git submodule.